src/components/Match/StyledMatch.jsx

```diff
@@ -68,6 +68,31 @@ export const StyledAghanimsBuffs = styled.div`
   }
 `
 
+export const StyledLevel = styled.div`
+  position: relative;
+  left: 2px;
+
+  .level_number {
+    position: absolute;
+    top: 6px;
```

Comment:
would be cool if we can center it automatically with properties rather than pixel values

Reply:
I'll try but I'm not sure if it's possible.